docs/tutorials/sending-and-receiving-messages-with-mqtt.md

```diff
@@ -50,9 +50,10 @@ Now we are able to send and receive messages through the broker (by default
 mosquitto uses port 1883). 
 
 This tutorial uses python scripts so we need to install one python library:
+.
+    pip3 install paho-mqtt
 
-    sudo easy_install paho-mqtt
-
+If not already available you can install 'pip3' by 'sudo apt-get install python3-pip'.
```

Comment:
Can this be moved before `pip` is needed? e.g. "If you have not already done so, you will need to..."